src/Orchard/ContentManagement/Utilities/LazyField.cs

```diff
@@ -11,7 +11,7 @@ public class LazyField<T> {
             set { SetValue(value); }
         }
 
-        public void Loader(Func<T, T> loader) {
+        public void Loader(Func<T> loader) {

```

Comment:
Hmmm... ok :)
Maybe should I create new LazyfField class and make obsolete both old ?
Agree ?